src/main/java/seedu/address/model/person/Attendance.java

```diff
@@ -31,7 +39,11 @@ public class Attendance {
     public Attendance(String attendance) {
         requireNonNull(attendance);
         checkArgument(isValidAttendance(attendance), MESSAGE_CONSTRAINTS);
-        this.time = LocalDate.parse(attendance, DTF);
+        try {
+            this.time = LocalDate.parse(attendance, DTF.withResolverStyle(ResolverStyle.STRICT));
+        } catch (DateTimeParseException dpe) {
+            throw new IllegalArgumentException(MESSAGE_INVALID_DATE);
+        }
```

Comment:
doesn't seem like this exception is properly handled afterwards?

Reply:
Done! Please take another look Ms Joanna

Reply:
yep looks better now